stdlib/public/core/SIMDVector.swift

```diff
@@ -1407,3 +1461,154 @@ where T: SIMD, T.Scalar: FloatingPoint {
+      (v / rhs, -lhs / (rhs * rhs) * v)
+    })
+  }
+}
+
+extension SIMD
+  where Self : Differentiable,
+        CotangentVector : SIMD,
+        Scalar : BinaryFloatingPoint & Differentiable,
+        Scalar.CotangentVector: BinaryFloatingPoint,
+        CotangentVector.Scalar == Scalar.CotangentVector {
+  @inlinable
+  static func _vjpAdd(
+    lhs: Scalar, rhs: Self
+    ) -> (Self, (CotangentVector) -> (Scalar.CotangentVector, CotangentVector)) {
+    return (lhs + rhs, { (v: CotangentVector) in
+      return (v.sum(), v)
+    })
+  }
+
+  @inlinable
+  static func _vjpSubtract(
+    lhs: Scalar, rhs: Self
+    ) -> (Self, (CotangentVector) -> (Scalar.CotangentVector, CotangentVector)) {
+    return (lhs + rhs, { (v: CotangentVector) in
+      return (v.sum(), -v)
+    })
+  }
+
+  @inlinable
+  static func _vjpAdd(
+    lhs: Self, rhs: Scalar
+    ) -> (Self, (CotangentVector) -> (CotangentVector, Scalar.CotangentVector)) {
+    return (lhs + rhs, { (v: CotangentVector) in
+      return (v, v.sum())
+    })
+  }
+
+  @inlinable
+  static func _vjpSubtract(
+    lhs: Self, rhs: Scalar
+    ) -> (Self, (CotangentVector) -> (CotangentVector, Scalar.CotangentVector)) {
+    return (lhs + rhs, { (v: CotangentVector) in
+      return (v, -v.sum())
+    })
+  }
+}
+
+extension SIMD
+  where Self : Differentiable,
+        CotangentVector : SIMD,
+        Scalar : BinaryFloatingPoint & Differentiable,
+        Self.CotangentVector == Self,
+        Scalar.CotangentVector == Scalar {
+  @inlinable
+  static func _vjpMultiply(
+    lhs: Self, rhs: Scalar
+    ) -> (Self, (CotangentVector) -> (CotangentVector, Scalar.CotangentVector)) {
+    return (lhs * rhs, { (v: CotangentVector) in
+      return (v * rhs, (v * lhs).sum())
+    })
+  }
+  
+  @inlinable
+  static func _vjpDivide(
+    lhs: Self, rhs: Scalar
+    ) -> (Self, (CotangentVector) -> (CotangentVector, Scalar.CotangentVector)) {
+    return (lhs / rhs, { (v: CotangentVector) in
+      (-lhs / (rhs * rhs) * v, (v / rhs).sum())
+    })
+  }
+  
+  @inlinable
+  static func _vjpMultiply(
+    lhs: Scalar, rhs: Self
+    ) -> (Self, (CotangentVector) -> (Scalar.CotangentVector, CotangentVector)) {
+    return (lhs * rhs, { (v: CotangentVector) in
+      return ((v * lhs).sum(), v * rhs)
+    })
+  }
+  
+  @inlinable
+  static func _vjpDivide(
+    lhs: Scalar, rhs: Self
+    ) -> (Self, (CotangentVector) -> (Scalar.CotangentVector, CotangentVector)) {
+    return (lhs / rhs, { (v: CotangentVector) in
+      ((v / rhs).sum(), -lhs / (rhs * rhs) * v)
+    })
+  }
+}
+
+extension SIMD
+  where Self : Differentiable,
+        CotangentVector : SIMD,
+        Scalar : BinaryFloatingPoint & Differentiable,
+        Scalar.CotangentVector : BinaryFloatingPoint,
+        CotangentVector == Self {
+  @usableFromInline
+  func _vjpSum() -> (Scalar, (Scalar.CotangentVector) -> CotangentVector) {
+    return (sum(), { (v: Scalar.CotangentVector)  in Self.init(repeating: Scalar(v)) })
```

Comment:
Omit `.init` and prefer the initialization syntax.

```suggestion
    return (sum(), { v in Self(repeating: Scalar(v)) })
```